source/MaterialXView/Viewer.h

```diff
@@ -86,8 +87,9 @@ class Viewer : public ng::Screen
     void drawScene2D();
 
     void setupLights(mx::DocumentPtr doc);
+    void setupUnitConverter(mx::DocumentPtr doc);
     void loadDocument(const mx::FilePath& filename, mx::DocumentPtr libraries);
-    void reloadShaders();
+    void reloadShaders(bool forceCreation);
```

Comment:
What's the proposed use case for forceCreation=true?  At the moment, I'm only seeing this method called from one place in the code, and forceCreation is set to false.